contents/docs/libraries/react/index.mdx

```diff
@@ -51,6 +51,7 @@ posthog.init(
   process.env.REACT_APP_PUBLIC_POSTHOG_KEY,
   {
     api_host: process.env.REACT_APP_PUBLIC_POSTHOG_HOST,
+    capture_pageview: 'history_change'
```

Comment:
No mention of what this does in this file? Maybe we need that somewhere?

Reply:
Yes, we just point there